src/filters/snowman.py

```diff
@@ -0,0 +1,29 @@
+from typing import List, Tuple
+
+
+class filter:
+    """Sample filter for ASCII operations"""
+
+    def __init__(self):
```

Comment:
you can inherit  a base filter class

Reply:
We can't, filters are dynamically loaded from a context where they can't directly access the main code to modify internal values/instantiate from classes.